src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -8849,6 +9134,8 @@ def fillna(
         axis: Optional[Axis] = None,
         limit: Optional[int] = None,
         downcast: Optional[dict] = None,
+        _filter_column_snowflake_quoted_id: Optional[str] = None,
+        _columns_to_ignore: Optional[list[bool]] = None,
```

Comment:
Is it possible to do this differently, i.e. not change the query compiler method's signature? I'm fine with doing this on internal methods that have leading underscores, but it'll be problematic if we do this for the public interface we want to inherit from `BaseQueryCompiler`.

Reply:
Done! Moved the code to a new underscored method with the additional arguments, and call the new underscored method from fillna without the additional arguments!